docs/modelserving/v1beta1/transformer/torchserve_image_transformer/README.md

```diff
@@ -70,7 +70,7 @@ class ImageTransformer(kserve.Model):
         return infer_request
 ```
 
-Please see the code example [here](https://github.com/kserve/kserve/tree/release-0.11/python/custom_transformer).
```

Comment:
guess we still need to point to the latest release, 0.14 or 0.13.
cc @yuzisun

Reply:
I wonder if there is way to use a release branch variable here and then resolve.